### PR TITLE
Make the installation howto guides consistent

### PR DESCRIPTION
**Pull request by keirthana**
* Use consistent structure and wording for the installation of Anbox Cloud Appliance on various types of cloud platforms. Ex: AWS, Azure, Google Cloud.
* Remove the redundant topic `Finalise.md`. Move any useful information in this topic that is not already covered to the `Install the appliance` tutorial.

### CODE REVIEW

File: howto/install-appliance/azure.md [hunk 1]
```diff
@@ -78,14 +78,21 @@ Azure will prompt you to download your private key before it starts creating the
...
 
 To do so, go to the resource page of your virtual machine and find its public IP address. Then use SSH to log on to the machine, using the user name `ubuntu` and the private key file that you downloaded during the creation of the virtual machine. For example:
 
     ssh -i Downloads/anbox-cloud-appliance_key.pem ubuntu@192.0.2.15
 
-## Install the appliance
+## Finish the installation
+
+Perform the following steps to finish the appliance installation on the virtual machine. If you are not already familiar with how to perform these steps, see the [tutorial on installing the appliance](https://discourse.ubuntu.com/t/22681) for detailed instructions.
+
```
Comment (adglkh) [inline howto/install-appliance/azure.md:92]: hmm, we need to attach Ubuntu Pro token before installing the appliance on azure, don't we?

File: howto/install-appliance/finalise.md [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-<a name="initialise"></a>
-## Initialise the appliance
-
-Invoke the initialisation process of the Anbox Cloud Appliance by entering the following command on your machine:
-
-    sudo anbox-cloud-appliance init
-
-You will be asked a few questions. If you don't want to make any specific changes, you can safely stay with the offered default answers.
-
-```bash
```
[the remark this answers is not in the document]
Comment (keirthana) [reply howto/install-appliance/finalise.md:27]: > Maybe we move this bash section to `tut/installing-appliance.md` page?

It's already moved (under Monitor the progress section)

File: howto/install-appliance/azure.md (hunk 1 above)
Comment (keirthana) [reply howto/install-appliance/azure.md:92]: > hmm, we need to attach Ubuntu Pro token before installing the appliance on azure, don't we?

It is part of the installation steps. We are redirecting the users to the tutorial if they don't already know/remember. In the tutorial, under "Install the appliance" section, there is a step for attaching Ubuntu Pro token.

File: howto/install-appliance/finalise.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adglkh) [reply howto/install-appliance/finalise.md:27]: I mean the whole bash block.
It seemed to me it was not there.

File: howto/install-appliance/azure.md (hunk 1 above)
Comment (adglkh) [reply howto/install-appliance/azure.md:92]: Yes, I see that. But I feel like it'd be better if we explicitly call that out here.
I don't have a strong opinion on this though.
let's keep as is if it looks like lengthy and redundant to you.

**Comment by adglkh** [comment]
LGTM

**Pull request merged**